dev-docs/bidders/iqx.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: bidder
+title: IQX
+description: IQX Bidder Adapter
+biddercode: iqx
+media_types: banner, video, native
+coppa_supported: true
+tcfeu_supported: false
+usp_supported: true
+prebid_member: false
+pbjs: false
+pbs: true
+schain_supported: true
+floors_supported: true
+multiformat_supported: will-bid-on-any
```

Comment:
An additional information, which is always helpful is the `safeframes_ok` meta data. From https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter#submitting-your-adapter